packages/plugins/documentation/server/services/helpers/build-component-schema.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+const cleanSchemaAttributes = require('./utils/clean-schema-attributes');
+const loopContentTypeNames = require('./utils/loop-content-type-names');
+const pascalCase = require('./utils/pascal-case');
+
+/**
+ * @decription Get all open api schema objects for a given content type
+ *
+ * @param {object} apiInfo
+ * @property {string} apiInfo.uniqueName - Api name | Api name + Content type name
+ * @property {object} apiInfo.attributes - Attributes on content type
+ * @property {object} apiInfo.routeInfo - The routes for the api
+ *
+ * @returns {object} Open API schemas
+ */
+const getAllSchemasForContentType = ({ routeInfo, attributes, uniqueName }) => {
```

Comment:
I don't think you take into account route prefixes. That could be great to improve the plugin

Reply:
Good idea but wouldn't that change be for the paths object? for example the user-permissions user prefix is `''` so the path should be
```
{
  paths: {
    '/users': {...} // vs /users-permissions/users
  }
}
```
Or maybe its for both `UsersResponse` vs `UsersPermissionsUsersResponse`?

Reply:
I kind of think `UsersPermissionsUserResponse` makes more sense because we should know it comes from a plugin.

Reply:
We can change only the paths and not the schema names I agree